src/Symfony/Component/Notifier/Bridge/Telegram/README.md

```diff
@@ -14,6 +14,27 @@ where:
  - `TOKEN` is your Telegram token
  - `CHAT_ID` is your Telegram chat id
```

Comment:
The option should be added to this list, similar to this: https://github.com/symfony/symfony/blob/7.3/src/Symfony/Component/Notifier/Bridge/AmazonSns/README.md#dsn-example